draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -866,7 +874,22 @@
    for change, rather than repeatedly fetching the whole collection.  An
    example of this is given in <xref target="examples-polling-status-changes"/>.</t>
 
+   	<section title="Extended View Trigger Status Resource Collections" anchor="polling-collections-extended">
+	<t>
+		If dCDN advertises support for extended status, uCDN MAY request to provide extended 
+		Trigger Collection, that embeds all Trigger Status resources in the collection in the collection itsef, 
```

Comment:
...uCDN MAY request _the_ extended Trigger Collection _view_, _which_ embeds all Trigger Status...
"in the collection" is repeated twice
itsef -> itself